src/server/qgis_map_serv.cpp

```diff
@@ -43,6 +44,16 @@ int fcgi_accept()
 
 int main( int argc, char *argv[] )
 {
+  if ( argc >= 2 )
```

Comment:
I know you can run this from the command line, but this is not really its designed usage: this is a FCGI binary to be executed from a wrapper process so this addition doesn't make much sense to me, anyway it won't hurt anyone so I'm not against it. It would probably be more useful to print QGIS version information in the server logs if it isn't already implemented.

Reply:
I understand @elpaso but I saw some people doing 
`ldd /usr/lib/cgi-bin/qgis_mapserv.fcgi | grep qgis`
to have the version number on a server.
So I thought to help them a little bit with this shortcut ;-)

It's quicker than having to know where the logs are etc.

Reply:
> I understand @elpaso but I saw some people doing
> `ldd /usr/lib/cgi-bin/qgis_mapserv.fcgi | grep qgis`

:smile: you could teach them to use `apt-cache` instead.

> to have the version number on a server.
> So I thought to help them a little bit with this shortcut ;-)
> 

Agreed.